docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.7"
```

Comment:
I don't think this file belongs in this library

Reply:
Removed